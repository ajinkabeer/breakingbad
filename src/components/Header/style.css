.header {
  left: 0;
  top: 0;
  width: 100%;
  height: 12rem;
  background-color: #0c6eef;
  padding: 0 2rem;
  display: flex;
  justify-content: center;
  align-items: center;
  box-sizing: border-box;
}

.header-title {
  color: #ffffff;
}
