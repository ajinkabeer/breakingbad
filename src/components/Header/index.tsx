import './style.css';

const Header = () => (
  <header className="header">
    <h1 className="header-title">Breaking Bad Characters</h1>
  </header>
);

export default Header;
