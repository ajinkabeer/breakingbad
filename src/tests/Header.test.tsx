import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import Header from '../components/Header'

describe('Header Component', () => {
    it('matches snapshot', () => {
        const { asFragment } = render(
            <Header />
        );
        expect(asFragment()).toMatchSnapshot();
    });
    it('renders the correct component', () => {
        const { getByText } = render(<Header />);
        getByText('Breaking Bad Characters');
    });
})