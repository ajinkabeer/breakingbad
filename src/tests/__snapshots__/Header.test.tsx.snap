// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <h1
      class="header-title"
    >
      Breaking Bad Characters
    </h1>
  </header>
</DocumentFragment>
`;
